### PR TITLE
refining content for core-data

### PR DESCRIPTION
**Pull request by meganmason**
added content to 03_database_preview; 2nd and 3rd examples are nearly complete. Need to finish the geojson part, and recap. Images would help too!

I will add content to notebook 02, and the final map figure in notebook 01

### CODE REVIEW

**Comment by JessicaS11** [comment]
@meganmason QC picked up a few spelling errors and tagged them. Other than fixing those, are you planning to add more to these tutorials in this PR (i.e. is it ready for review)?

**Comment by meganmason** [comment]
hi @JessicaS11, yes - still adding content, so it's not ready for a final review. I know I'm past the preferred deadline. (Unfortunately, I'm now limited to short breaks while I attend a 2-day conference).  Thanks for checking in!

**Comment by micahjohnson150** [comment]
@meganmason yesterday @scottyhq was having issues with a reference to `ImageData.surveyors` in your notebook. So to move forward he temporarily disabled one of your notebooks. Looks like you fixed the reference to surveyors here but I think it wont render without removing it from the [_config.yml ](https://github.com/snowex-hackweek/website2022/blob/45ad291cbb074503d048bf7f5422de177415ce70/book/_config.yml#L43)

**Comment by meganmason** [comment]
okay, team! This is ready. There might be some requested changes, a typo, or a check that doesn't pass, but I'm finished with the content. I'll watch the checks and see which ones I can manage.

**Comment by meganmason** [comment]
thanks @micahjohnson150 for the tip on the _config.yml file. I removed it and pushed anew.

**Comment by meganmason** [comment]
How does one handle a supposed spell check error that is not an error? In my case it's a table with a site name "BA flux tower" and the request is to change 'BA' to 'BY' or 'BE', which is not correct.

**Comment by jomey** [comment]
> How does one handle a supposed spell check error that is not an error? In my case it's a table with a site name "BA flux tower" and the request is to change 'BA' to 'BY' or 'BE', which is not correct.

Add the “BA” at the end of this list:
https://github.com/snowex-hackweek/website2022/blob/e4c2652549353647689fbc22aedc77706e95ee88/.github/workflows/qaqc.yaml#L36

**Comment by meganmason** [comment]
thanks @jomey ! I changed the file on my branch and pushed it up. Could you remind me how to "deploy" the preview?

File: book/tutorials/core-datasets/01_data_coverage.ipynb [hunk 1]
```diff
@@ -48,37 +75,63 @@
...
-   "id": "4961f594-ed9a-4394-8683-19c7f3ee27be",
+   "id": "4129c1af-dc53-4597-9015-ce2affc6b32a",
    "metadata": {},
    "source": [
-    "<img src='content/01_where-has-snowex-been.png'>"
+    "### What's with the IOP versus the TS?\n",
+    "These are two different sampling strategies, but result in the same types of measurements and data set products. Depending on the  research application it may not matter at all which you choose to work with, or even combine! The important thing to grasp is the difference in spatial and temporal extent of the campaign periods. If the sampling protocols or data set products change over time, it is for the sake of improvement. When possible, we aim to keep things consistent to continue to build a legacy data set. \n",
+    "\n",
+    "<figure>\n",
+    "<img src='content/01_campaign_venn_diagram.jpg'>\n",
```
Comment (scottyhq) [inline book/tutorials/core-datasets/01_data_coverage.ipynb]:
```suggestion
    "<img src='./content/01_campaign_venn_diagram.jpg'>\n",
```
i think you need relative paths

**Review by scottyhq — CHANGES_REQUESTED** [review]
@meganmason the preview is automatically updated with each commit, so your content is here https://deploy-preview-73--snowex2022.netlify.app/tutorials/core-datasets/index.html . The images are not rendering currently, I think the paths need to be relative to the notebook (see comment)

**Comment by meganmason** [comment]
made all images have relative file paths, but the preview still doesn't seem to be rendering them?

**Comment by meganmason** [comment]
The pattern appears to be all images I have a figure capture for are NOT rendering, the few images without figure captions are okay.

**Comment by meganmason** [comment]
Images all render and required checks pass. I think we're good to merge by the AM

**Review by scottyhq — APPROVED** [review]

**Pull request merged**